packs/tests/actions/chains/test_key_triggers.yaml

```diff
@@ -91,6 +91,7 @@ chain:
                 object:
                     name: a
                     value: b
+                    uid: "key_value_pair:a"
```

Comment:
It looks like that in some cases it might be useful to also have "non-strict" object equals. Anyways.